<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CorrectionTab</class>
 <widget class="QWidget" name="CorrectionTab">
    <property name="styleSheet">
   <string notr="true">QGroupBox {
border: 1px solid grey;border-radius: 10px;margin-top: 1ex; margin-right: 0ex
}
QGroupBox:title {
                           subcontrol-origin: margin;
                           padding: 0 3px;
                           subcontrol-position: top center;
                           padding-top: 0px;
                          padding-bottom: 0px;
                           padding-right: 10px;
                            color: rgb(56, 56, 56)
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">

   <!-- Load Data Section -->
   <item>
    <widget class="QGroupBox" name="groupBox_loadData">
     <property name="title">
      <string>Load Data</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_load">
      <item>
       <widget class="FileFinderWidget" name="finder_corr" native="true"/>
      </item>
        <item>
   <widget class="QPushButton" name="btn_loadFiles">
    <property name="text">
     <string>Load Files</string>
    </property>
   </widget>
  </item>
      <item>
       <widget class="QTableWidget" name="table_loaded_data"/>
      </item>
      <item>
 <layout class="QHBoxLayout" name="horizontalLayout_tableButtons">
  <item>
   <spacer name="spacer_left">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="sizeHint" stdset="0">
     <size>
      <width>40</width>
      <height>20</height>
     </size>
    </property>
   </spacer>
  </item>
    <item>
   <widget class="QPushButton" name="btn_deleteSelected">
    <property name="text">
     <string>Delete Selected Files</string>
    </property>
   </widget>
  </item>
  <item>
   <widget class="QPushButton" name="btn_selectAll">
    <property name="text">
     <string>Select All Files</string>
    </property>
   </widget>
  </item>
  <item>
   <spacer name="spacer_right">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="sizeHint" stdset="0">
     <size>
      <width>40</width>
      <height>20</height>
     </size>
    </property>
   </spacer>
  </item>
 </layout>
</item>
     </layout>
    </widget>
   </item>
 <!-- Sample Section -->
 <item>
 <widget class="QGroupBox" name="groupBox_sample">
  <property name="title">
   <string>Set Sample Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_sample">
<!-- Orientation Section -->
   <item>
    <widget class="QGroupBox" name="groupBox_orientation">
     <property name="title">
      <string>Orientation</string>
     </property>
 <property name="styleSheet">
  <string notr="true">
   QGroupBox {
    border: 1px solid #ccc;
    border-radius: 4px;
    margin-top: 10px;
   }
   QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 5px;
    font-weight: normal;
    color: #555;
   }
  </string>
 </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QPushButton" name="btn_setOrientation">
        <property name="text">
         <string>Set Single Orientation</string>
        </property>
       </widget>
      </item>
<item>
  <widget class="QLabel" name="label_or">
    <property name="text">
      <string>or</string>
    </property>
    <property name="alignment">
      <set>Qt::AlignCenter</set>
    </property>
    <property name="styleSheet">
      <string notr="true">color: grey; font-style: italic;</string>
    </property>
  </widget>
</item>
      <item>
       <widget class="FileFinderWidget" name="finder_orientation_file" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_loadOrientation">
        <property name="text">
         <string>Load Orientation File</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <!-- Sample Shape Section -->
   <item>
    <widget class="QGroupBox" name="groupBox_shape">
     <property name="title">
      <string>Sample Shape</string>
     </property>
 <property name="styleSheet">
  <string notr="true">
   QGroupBox {
    border: 1px solid #ccc;
    border-radius: 4px;
    margin-top: 10px;
   }
   QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 5px;
    font-weight: normal;
    color: #555;
   }
  </string>
 </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QPushButton" name="btn_loadSampleShape">
        <property name="text">
         <string>Load Shape onto single WS</string>
        </property>
       </widget>
      </item>
<item>
  <widget class="QLabel" name="label_or">
    <property name="text">
      <string>or</string>
    </property>
    <property name="alignment">
      <set>Qt::AlignCenter</set>
    </property>
    <property name="styleSheet">
      <string notr="true">color: grey; font-style: italic;</string>
    </property>
  </widget>
</item>
      <item>
       <widget class="QPushButton" name="btn_setSampleShape">
        <property name="text">
         <string>Set Shape onto single WS</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <!-- Sample Material Section -->
   <item>
    <widget class="QGroupBox" name="groupBox_material">
     <property name="title">
      <string>Sample Material</string>
     </property>
 <property name="styleSheet">
  <string notr="true">
   QGroupBox {
    border: 1px solid #ccc;
    border-radius: 4px;
    margin-top: 10px;
   }
   QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 5px;
    font-weight: normal;
    color: #555;
   }
  </string>
 </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QPushButton" name="btn_setSampleMaterial">
        <property name="text">
         <string>Set Sample Material</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <!-- Apply to All Section -->
   <item>
    <widget class="QGroupBox" name="groupBox_applyAll">
     <property name="title">
      <string>Apply to All Workspaces</string>
     </property>
 <property name="styleSheet">
  <string notr="true">
   QGroupBox {
    border: 1px solid #ccc;
    border-radius: 4px;
    margin-top: 10px;
   }
   QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 5px;
    font-weight: normal;
    color: #555;
   }
  </string>
 </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QComboBox" name="combo_workspaceList"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_copySampleToAll">
        <property name="text">
         <string>Copy Full Sample Environment to All</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

     </layout>
 </widget>
</item>

   <!-- Absorption Correction Section -->
   <item>
    <widget class="QGroupBox" name="groupBox_absorption">
     <property name="title">
      <string>Absorption Correction</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="check_absorption">
        <property name="text">
         <string>Include Absorption Correction</string>
        </property>
       </widget>
      </item>
       <item>
           <widget class="QWidget" name="widget_absorptionContainer">
             <layout class="QVBoxLayout">
      <item>
       <widget class="QComboBox" name="combo_shapeMethod">
        <item>
         <property name="text">
          <string>4mmCube</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Custom Shape</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="FileFinderWidget" name="finder_gauge_vol" native="true"/>
        </item>
       </layout>
      </item>
                        <!-- Attenuation Table Section -->
   <item>
    <widget class="QGroupBox" name="groupBox_attenuationTable">
     <property name="title">
      <string>Tabulate Attenuation Coefficient</string>
     </property>
        <property name="styleSheet">
  <string notr="true">
   QGroupBox {
    border: 1px solid #ccc;
    border-radius: 4px;
    margin-top: 10px;
   }
   QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 5px;
    font-weight: normal;
    color: #555;
   }
  </string>
 </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="check_attenTab">
        <property name="text">
         <string>Create Attenuation Value Table</string>
        </property>
       </widget>
      </item>
      <item>
          <widget class="QWidget" name="widget_attenuationTableContainer">
              <layout class="QHBoxLayout">
                  <item>
       <widget class="QLabel" name="label_value">
        <property name="text">
         <string>Evaluation Point: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_evalVal"/>
      </item>
                  <item>
       <widget class="QLabel" name="label_units">
        <property name="text">
         <string>Units: </string>
        </property>
       </widget>
      </item>
                  <item>
  <widget class="QComboBox" name="combo_Units">
    <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
  </widget>
</item>
              </layout>
          </widget>
      </item>

     </layout>
    </widget>
   </item>
         </layout>
    </widget>
   </item>

     </layout>
    </widget>
   </item>

   <!-- Divergence Correction Section -->
   <item>
    <widget class="QGroupBox" name="groupBox_divergence">
     <property name="title">
      <string>Beam Divergence Correction</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="check_divergence">
        <property name="text">
         <string>Include Beam Divergence Correction</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_divHorz">
        <property name="text">
         <string>Incident Horizontal:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="line_divHorz"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_divVert">
        <property name="text">
         <string>Incident Vertical:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="line_divVert"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_detHorz">
        <property name="text">
         <string>Detector Horizontal:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="line_detHorz"/>
      </item>
     </layout>
    </widget>
   </item>


   <!-- Exec Button -->
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="btn_applyCorrections">
       <property name="text">
        <string>Apply Corrections</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileFinderWidget</class>
   <extends>QWidget</extends>
   <header>mantidqt.widgets.filefinderwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
